smoke-tests/docker-compose.yml

```diff
@@ -5,6 +5,8 @@ x-env-base: &env_base
   HONEYCOMB_API_KEY: bogus_key
   HONEYCOMB_DATASET: bogus_dataset
   HONEYCOMB_METRICS_DATASET: bogus_dataset
+  OTEL_METRIC_EXPORT_INTERVAL: 5000
```

Comment:
I tried going down to 2500 but it started failing tests because the value showed up too many times in the output. These seemed to be ensure a more consistent pass.